src/groups/mqb/mqbblp/mqbblp_clusterqueuehelper.cpp

```diff
@@ -4611,43 +4452,6 @@ void ClusterQueueHelper::openQueue(
 
     QueueContextMapIter queueContextIt = d_queues.find(uriKey);
 
-    // NOTE: See TBD in 'onGetDomainDispatched': if the request comes from a
```

Comment:
To be honest, I am quite unfamiliar with logic surrounding 'clientIdentity'. Are you sure it is safe to remove this code block?